### DISCUSSION TITLE
Is upsert supported?

### DISCUSSION THREAD

**Discussion by danechitoaie** [piccolo-orm/piccolo#815] [Q&A] [answered]
Is upsert supported?

**Comment by sinisaos** [comment]
@danechitoaie For now there is no api for upserting but here is [PR](https://github.com/piccolo-orm/piccolo/pull/798) for that. PR works but there is disscussion on how api would be. Like in PR with `on_conflict` as argument in insert or `on_conflict` should be a separate method.

**Comment by dantownsend** [comment, marked as answer]
Upsert is supported now - https://piccolo-orm.readthedocs.io/en/latest/piccolo/query_clauses/on_conflict.html
